app/src/main/java/com/example/dine_and_donate/HomeFragments/NotificationsAdapter.java

```diff
@@ -75,6 +76,8 @@ public void onCancelled(@NonNull DatabaseError databaseError) {
 
             }
         });
+        holder.mNotifiedAt.setText("2 m ago");
+        //holder.mNotifiedAt.setText(DateUtils.getRelativeTimeSpanString(notification.getCreatedAt(), System.currentTimeMillis(),DateUtils.M).toString());
```

Comment:
??

Reply:
was trying to see how it would look like on the layout before  I decided to populate the text view the the info from the database